jsonnet/telemeter/rules.libsonnet

```diff
@@ -171,23 +171,18 @@
               // OpenShift Cluster effective cores for subscription usage.
               // This counts both worker nodes and, when the control plane is schedulable, control plane nodes.
               // Only CoreOS nodes are counted.
-              // 1. x86_64 nodes that show hyperthreading in the telemetry have an accurate cores value in node_role_os_version_machine:cpu_capacity_cores:sum.
-              // 2. x86_64 nodes that do not show hyperthreading need the cores value adjusted to account for 2 threads per core (* 0.5).
-              // 3. Other CPU architectures are assumed to have accurate values in node_role_os_version_machine:cpu_capacity_cores:sum.
+              // 1. x86_64 nodes need the cores value adjusted to account for 2 threads per core (* 0.5).
+              // 2. Other CPU architectures are assumed to have accurate values in cluster:capacity_cpu_cores:sum.
               record: 'cluster:capacity_effective_cpu_cores',
```

Comment:
since the recording expression only uses `cluster:capacity_cpu_cores:sum`, shouldn't we also modify  `cluster:cpu_capacity_cores:_id` to use the same metric? It would also simplify the test inputs since the tests would properly generate `cluster:capacity_cpu_cores:_id`.